### PR TITLE
query parameter for hiding completed jobs was added 

### PR DESCRIPTION
**Pull request by ssuvorov-fls**
fixes https://github.com/OHDSI/Atlas/issues/539

### CODE REVIEW

File: src/main/java/org/ohdsi/webapi/job/NotificationController.java [hunk 1]
```diff
@@ -31,8 +33,9 @@
     @Path("/")
     @Produces(MediaType.APPLICATION_JSON)
     @Transactional(readOnly = true)
-    public List<JobExecutionResource> list() {
-        return service.findLast10().stream().map(this::toDTO).collect(Collectors.toList());
+    public List<JobExecutionResource> list(
+            @QueryParam("hideCompleted") @DefaultValue("false") boolean hideCompleted) {
```
Comment (wivern) [inline src/main/java/org/ohdsi/webapi/job/NotificationController.java]: According to general REST recommendations, it's better to use a `snake_case` or a `spinal-case`. To be consistent pls change query parameter to `snake_case` as one is widely used in the WebAPI. 
Second approach is the: why the API method is limited to just show/hide jobs with completed status. Why not to resolve list of statuses that should be filtered out?

File: src/main/java/org/ohdsi/webapi/job/NotificationController.java [hunk 2]
```diff
@@ -31,8 +35,15 @@
...
     @Produces(MediaType.APPLICATION_JSON)
     @Transactional(readOnly = true)
-    public List<JobExecutionResource> list() {
-        return service.findLast10().stream().map(this::toDTO).collect(Collectors.toList());
+    public List<JobExecutionResource> list(
+            @QueryParam("hide_statuses") String hideStatuses) {
+        List<BatchStatus> statuses = new ArrayList<>();
+        if (StringUtils.isNotEmpty(hideStatuses)) {
+            for (String status : hideStatuses.split(",")) {
+                statuses.add(BatchStatus.valueOf(status));
```
Comment (wivern) [inline src/main/java/org/ohdsi/webapi/job/NotificationController.java]: Could you add some check in a case when status value is odd and cannot be converted to a `BatchStatus`?

File: src/main/java/org/ohdsi/webapi/job/NotificationController.java (hunk 1 above)
Comment (ssuvorov-fls) [reply src/main/java/org/ohdsi/webapi/job/NotificationController.java]: done

File: src/main/java/org/ohdsi/webapi/job/NotificationController.java (hunk 2 above)
Comment (ssuvorov-fls) [reply src/main/java/org/ohdsi/webapi/job/NotificationController.java]: try/catch was added

**Pull request merged**